setup.py

```diff
@@ -1028,11 +1028,15 @@ def main():
         'opt-einsum': ['opt-einsum>=3.3']
     }
     if platform.system() == 'Linux':
-        triton_pin_file = os.path.join(cwd, ".github", "ci_commit_pins", "triton.txt")
-        if os.path.exists(triton_pin_file):
-            with open(triton_pin_file) as f:
-                triton_pin = f.read().strip()
-                extras_require['dynamo'] = ['pytorch-triton==2.0.0+' + triton_pin[:10], 'jinja2']
+        pytorch_extra_install_requirements = os.getenv("PYTORCH_EXTRA_INSTALL_REQUIREMENTS", "")
+        if 'triton' in pytorch_extra_install_requirements:
+            extras_require['dynamo'] = ['pytorch-triton==2.0.0', 'jinja2']
```

Comment:
Should this be triton==2.0.0 instead of pytorch-triton==2.0.0?  
Is there any connection to PYPI's triton or we are just doing version upgrade from 2.0.0+gitcommit to 2.0.0?

Reply:
I think you are right!

Reply:
To be clear, my question is where is the pytorch-triton==2.0.0 package?

Reply:
Basically we need to point to here: https://pypi.org/project/triton/